troposphere/static/js/components/projects/detail/resources/ProjectDetails.react.js

```diff
@@ -161,7 +121,8 @@ define(function (require) {
         selectedResource = this.state.selectedResource,
         isButtonBarVisible;
 
-      if (!projectInstances || !projectImages || !projectExternalLinks || !projectVolumes) return <div className="loading"></div>;
+      if (!projectInstances || !projectImages || !projectExternalLinks || !projectVolumes) 
+          return <div className="loading"></div>;
```

Comment:
:+1: 

I'd be fine with `{ ... }` around this too. Not sure if you are in favor of that style or not. It's not super important either way.